### PR TITLE
Release 1.16.0

### PR DESCRIPTION
**Pull request by paprikati**
* uses new instalment schedules schema (with two different create methods)
* adds 'count' field to subscriptions response

### CODE REVIEW

File: gocardless_pro/resources/subscription.py [hunk 1]
```diff
@@ -123,6 +128,8 @@ def upcoming_payments(self):
 
   
 
+
```
Comment (paprikati) [inline gocardless_pro/resources/subscription.py]: fun

File: tests/fixtures/bank_details_lookups.json [hunk 2]
```diff
@@ -3,6 +3,6 @@
     "method": "POST",
     "path_template": "/bank_details_lookups",
     "url_params": [],
-    "body": {"bank_details_lookups":{"available_debit_schemes":["bacs"],"bank_name":null,"bic":null}}
+    "body": {"bank_details_lookups":{"available_debit_schemes":["bacs"],"bank_name":"NATWEST BANK PLC","bic":"BARCGB22XXX"}}
```
Comment (dmagliola) [inline tests/fixtures/bank_details_lookups.json]: I love unrelated changes 😹

Comment (paprikati) [reply tests/fixtures/bank_details_lookups.json]: the json fixtures are generated by something randomised, so its never the same :(

**Comment by paprikati** [comment]
> This seems to be adding more stuff than the PHP one.

I believe it's just about whether the json fixtures are auto-generated? Are there other changes in this diff other than JSON and the instalment schedules + subscriptions 'count' ?

File: tests/fixtures/bank_details_lookups.json (hunk 2 above)
Comment (dmagliola) [reply tests/fixtures/bank_details_lookups.json]: 🤦‍♂ 😭

**Comment by dmagliola** [comment]
nope, just the autogeneration of fixtures.
🚢
